kolibri/plugins/coach/assets/src/views/reports/ReportsLessonLearnerPage.vue

```diff
@@ -47,7 +47,10 @@
             <td>
               <StatusSimple :status="tableRow.statusObj.status" />
             </td>
-            <td><TimeDuration :seconds="tableRow.statusObj.time_spent" /></td>
+            <td><TimeDuration
+              v-if="tableRow.statusObj.status !== STATUSES.notStarted"
+              :seconds="tableRow.statusObj.time_spent"
+            /></td>
```

Comment:
same as above - `undefined`

Also super minor style note, usually I push multi-line tags to their own lines, ie:

```html
          <td>
            <TimeDuration
              v-if="tableRow.statusObj.status !== STATUSES.notStarted"
              :seconds="tableRow.statusObj.time_spent"
            />
          </td>
```